app/code/Magento/Quote/Model/GuestCart/GuestBillingAddressManagement.php

```diff
@@ -44,7 +43,7 @@ public function __construct(
      */
     public function assign($cartId, \Magento\Quote\Api\Data\AddressInterface $address, $useForShipping = false)
     {
-        /** @var $quoteIdMask QuoteIdMask */
+        /** @var $quoteIdMask QuoteIdMaskFactory */
```

Comment:
I believe the original code comment is correct here. It describes the type of the $quoteIdMask. QuoteIdMaskFactor::create() will return an instance of QuoteIdMask and it will remain QuoteIdMask when load is called on it.

Reply:
You're right, thanks!